02 Algorithm Reference/14 Reality Modelling/09 Volatility Model.html

```diff
@@ -0,0 +1,7 @@
+<meta name="tag" content="trading and orders">
+<p>
+The volatility model is a property of a security. Exactly how volatility is calculated varies a lot between strategies so we've provided an override point here. Volatility models get updated with data each time step and are expected to be updated immediately. This is primarily required for options backtesting.
+</p>
+<p>
+	Volatility models implement the $[IVolatilityModel,T:QuantConnect.Securities.IVolatilityModel] interface. We default to the NullVolatilityModel which returns 0 volatility at all times. As a helper we also provide the $[RelativeStandardDeviationVolatilityModel,T:QuantConnect.Securities.RelativeStandardDeviationVolatilityModel] which calculates the volatility based on standard deviation.
```

Comment:
`$[NullVolatilityModel, T:QuantConnect.Securities.VolatilityModel.NullVolatilityModel]`

Reply:
`T:QuantConnect.Securities.VolatilityModel.NullVolatilityModel` This doesn't work
https://www.quantconnect.com/services/inspector?type=T:QuantConnect.Securities.VolatilityModel.NullVolatilityModel

Reply:
This should be only: `T:QuantConnect.Securities.VolatilityModel`

`NullVolatilityModel` is a private class, `VolatilityModel.Null` is a static field returning a null instance.